### DISCUSSION TITLE
Source of quotation for SSH connection between manager and nodes

### DISCUSSION THREAD

**Discussion by jnikodemus** [Security-Onion-Solutions/securityonion#4281] [Unsupported Versions] [answered]
Hello Community!
The documentation says that the initial ssh-connection between manager and node is done using the account soremote. If I am right, node and manager are then doing a key exchange to establish the ssh-tunnel without sending the password again. My problem is that I can't find any source of that exchange or the afterwards used ssh-tunnel in the documentation. Does anyone know if there is a reliable source that I can quote in an official document?

**UPDATE:**
I found that there is no ssh-tunnel between manager and node after the initial configuration.
Is there any connection between manager and nodes that is not encrypted? I would like to establish such a connection over the internet and need to know if a VPN is needed.

**Comment by weslambert** [comment, marked as answer]
There is no persistent SSH tunnel after setup.  All other communication happens over the various other application ports, using TLS where appropriate.
